.github/workflows/build_and_test_workflow.yml

```diff
@@ -47,6 +47,14 @@ jobs:
         run: |
           git config --global core.autocrlf false
 
+      - name: Configure pagefile size (Windows only)
```

Comment:
Can workflow files take comments? An explanation here might be helpful for if/when GitHub improves their Windows infra and we can remove these.